docs/stories/Inputs/Combobox.stories.tsx

```diff
@@ -76,8 +48,64 @@ export const Controlled = {
       </Combobox>
     );
   },
+};
+
+export const Basic = {
+  ...Template,
```

Comment:
because this template is a story which contains render component, and by spreading we are passing additional/variable props on the story eg: args, parameters